data/json/items/tool/tailoring.json

```diff
@@ -347,7 +348,7 @@
     "qualities": [ [ "CUT", 2 ], [ "BUTCHER", 5 ], [ "FABRIC_CUT", 2 ] ],
     "flags": [ "BELT_CLIP", "SHEATH_KNIFE", "ALLOWS_BODY_BLOCK" ],
     "weapon_category": [ "SHIVS" ],
-    "melee_damage": { "stab": 10 }
+    "melee_damage": { "cut": 10, "stab": 5 }
```

Comment:
Same as the other scissors, it's just stab, and it's a pretty terrible weapon so it definitely doesn't need more damage added.